test/lib/y2firewall/clients/installation_finish_test.rb

```diff
@@ -62,6 +62,33 @@
         expect(subject.write).to eq true
       end
     end
+
+    context "when running in AutoYaST" do
```

Comment:
Wanted to ask about updating the unit test but you already did ... so, thnx ;)

Reply:
welcome ... I can read you as a book ;-)